build.sbt

```diff
@@ -26,6 +28,41 @@ def common: Seq[Setting[_]] =
     Compile / javacOptions ++= Seq("-encoding", "UTF-8"),
     Compile / scalacOptions ++= Seq("-encoding", "UTF-8", "-deprecation"))
 
+lazy val devTools = project
+  .in(file("devtools"))
```

Comment:
This is just the part to start the containers. Potentially usable from Protobuf SDKs as well.